libraries/rush-lib/src/api/SubspacesConfiguration.ts

```diff
@@ -24,6 +24,7 @@ interface ISubspacesConfigurationJson {
   subspacesEnabled: boolean;
   splitWorkspaceCompatibility?: boolean;
   preventSelectingAllSubspaces?: boolean;
+  preventFilteredInstallSubspaces?: string[];
   subspaceNames: string[];
```

Comment:
This file would be more readable if it contained a list of subspace objects with properties instead of two lists of names.

Reply:
We considered such a design originally, but at the time, the thinking was that each subspace's configuration should go in `comon/config/my-subspace/*.json` rather than in this file.  Like for example does this really belong in `pnpm-config.json`?